TextDumpOutputer.h

```diff
@@ -7,19 +7,24 @@
 namespace cce::tf {
 class TextDumpOutputer : public OutputerBase {
```

Comment:
There are a lot of overrides marked `final`.  Would it be easier to just mark the entire class as `final`?

```cpp
class TextDumpOutputer final : public OutputBase { ... };
```

Reply:
done